src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/service/publication/PublicationRetrievalServiceTest.java

```diff
@@ -11,11 +11,11 @@
 
 @ActiveProfiles("integration-basic")
 @SpringBootTest
-class ArtefactServiceTest extends IntegrationBasicTestBase {
+class PublicationRetrievalServiceTest extends IntegrationBasicTestBase {
     private static final String PAYLOAD_LIMIT_MESSAGE = "Payload limit result does not match";
 
     @Autowired
-    ArtefactService artefactService;
+    PublicationRetrievalService artefactService;
```

Comment:
rename artefactService to publicationRetrievalService

Reply:
changed to publicationRetrievalService